proto/io.prometheus.client.rs

```diff
@@ -0,0 +1,94 @@
+#[derive(Clone, PartialEq, ::prost::Message)]
```

Comment:
I think by following https://docs.rs/prost-build/0.7.0/prost_build/  we can use the `include!` macro to include the generated file at compile time. Then we don't need to include the generated file in the source code any more.

Reply:
Good point. Thanks.